pom.xml

```diff
@@ -356,6 +356,17 @@
         <enabled>false</enabled>
       </snapshots>
     </repository>
+    <repository>
```

Comment:
The new version of `RoaringBitmap` requires configuring additional repositories:

- https://github.com/RoaringBitmap/RoaringBitmap?tab=readme-ov-file#usage-within-a-maven-project
![image](https://github.com/user-attachments/assets/14f01b07-8823-4e7a-b6c8-749519918da9)

- https://github.com/RoaringBitmap/RoaringBitmap?tab=readme-ov-file#2-using-github-packages

![image](https://github.com/user-attachments/assets/e399f433-57c7-4bad-9771-e175d1102b0a)

The second method requires additional configuration of a GitHub token, so Method One was chosen here.